webpush_channels/tests/test_views_channel.py

```diff
@@ -124,3 +124,15 @@ def test_push_notification_can_take_a_payload(self):
             webpusher_mock.assert_called_with(self.subscription)
             webpusher_mock.return_value.send.assert_called_with(
                 data=MINIMALIST_PAYLOAD['data'], ttl=15)
+
+    def test_invalid_encryption_keys_shows_error(self):
+        CHANGED_SUBSCRIPTION = MINIMALIST_SUBSCRIPTION
+        CHANGED_SUBSCRIPTION['data']['keys']['p256dh'] = 'y'
+
+        self.app.post_json(self.subscription_url,
+                           CHANGED_SUBSCRIPTION,
+                           headers=self.headers)
```

Comment:
The 400 is supposed to happen here because that's the CHANGED_SUBSCRIPTION which is invalid.